easybuild/easyconfigs/g/gmsh/gmsh-3.0.6-foss-2017b-Python-2.7.14.eb

```diff
@@ -19,15 +19,17 @@ builddependencies = [
     ('SWIG', '3.0.12', versionsuffix),
 ]
 
+local_pymajmin = '2.7'
 dependencies = [
-    ('Python', '2.7.14'),
+    ('Python', local_pymajmin + '.14'),
     ('PETSc', '3.8.3', '-downloaded-deps'),
     ('SLEPc', '3.8.3'),
 ]
 
 separate_build_dir = True
 
 configopts = '-DENABLE_FLTK=0 -DENABLE_WRAP_PYTHON=ON -DENABLE_METIS=1'
```

Comment:
@jhein32 Please add back the extra space at the end, it's needed to separate `-DENABLE_METIS=1` from `-DPython...`

Reply:
Thanks.  I understood this and was doing it, while you were reviewing things.  I think all requests are done.  Let me know if there is stuff I overlooked.